fe/fe-core/src/main/java/org/apache/doris/nereids/util/ExpressionUtils.java

```diff
@@ -89,51 +87,36 @@ public static Expression or(List<Expression> expressions) {
      * Use AND/OR to combine expressions together.
      */
     public static Expression combine(NodeType op, List<Expression> expressions) {
-
         Objects.requireNonNull(expressions, "expressions is null");
 
         if (expressions.size() == 0) {
-            if (op == NodeType.AND) {
-                return new Literal(true);
-            }
-            if (op == NodeType.OR) {
-                return new Literal(false);
-            }
-        }
-
-        if (expressions.size() == 1) {
+            return new Literal(op == NodeType.AND);
```

Comment:
If u do that, u need add a check at the top of this function to check `NodeType` is either `AND` or `OR`